presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1372,7 +1372,6 @@ public PhysicalOperation visitUnnest(UnnestNode node, LocalExecutionPlanContext
             }
             if (ordinalitySymbol.isPresent()) {
                 outputMappings.put(ordinalitySymbol.get(), channel);
-                channel++;
```

Comment:
I prefer to leave these in, so if someone adds code after this section, it won't fail.